examples/step-6/doc/intro.dox

```diff
@@ -3,12 +3,105 @@
 
 @dealiiVideoLecture{15,16,17,17.25,17.5,17.75}
 
-The main emphasis in this example is the handling of locally refined
-grids. The approach to adaptivity chosen in deal.II is to use grids in which
+This program is finally about one of the main features of deal.II:
+the use of adaptively (locally) refined meshes. The program is still
+based on step-4 and step-5, and as you will see, it does not actually
+take very much code to enable adaptivity. Indeed, while we do a great
+deal of explaining, adaptive meshes can be added to an existing program
+with barely a dozen lines of additional code. The program shows what
+these lines are, as well as another important ingredient of adaptive
+mesh refinement (AMR): a criterion that can be used to determine whether
+it is necessary to refine a cell because the error is large on it,
+whether the cell can be coarsened because the error is particularly
+small on it, or whether we should just leave the cell as it is. We
+will discuss all of these issues in the following.
+
+
+<h3> How adaptively refined meshes look like </h3>
+
+There are a number of ways how one can adaptively refine meshes. The
+basic structure of the overall algorithm is always the same and consists
+of a loop over the following steps:
+- Solve the PDE on the current mesh;
+- Estimate the error on each cell using some criterion that is indicative
+  of the error;
+- Mark those cells that have large errors for refinement, mark those that have
+  particularly small errors for coarsening, and leave the rest alone;
+- Refine and coarsen the cells so marked to obtain a new mesh;
+- Repeat the steps above on the new mesh until the overall error is
+  sufficiently small.
+
+For reasons that are probably lost to history (maybe that these functions
+used to be implemented in FORTRAN, a language that does not care about
+whether something is spelled in lower or UPPER case letters, with programmers
+often choosing upper case letters habitually), the loop above is often
```

Comment:
IIRC correctly a lot of IBM computers back then used 6 bit encodings that did not allow for lower case letters (good old BCD)

Reply:
Yes. I didn't add this but can if you want.

Reply:
Well, what you wrote is still correct, so its not a big deal.

Reply:
(so we can keep it as-is)